src/nimib/themes.nim

```diff
@@ -20,6 +20,7 @@ const head* = """
   {{{favicon}}}
   <meta content="text/html; charset=utf-8" http-equiv="content-type">
   <meta content="width=device-width, initial-scale=1" name="viewport">
+  <meta content="nimib {{{version}}}" name="generator">
```

Comment:
It is not going to matter for this but I would go with the two brackets only version `{{version}}`. The difference is that the triple bracket one (or equivalently `{{&not_escaped}}` and I actually started to prefer this since it is more evident it is different but nimib code base still mostly uses triple braces) does not escape content so that you can add valid html.

Reply:
Okay, I changed it to use only two brackets.